### ISSUE TITLE
invalid literal

### ISSUE THREAD

**Issue by solsimpais** [JoinMarket-Org/joinmarket#390]
Getting this error message:
```
 1: {},
 2: {},
 3: {}}
2016-01-14 10:21:55,203 [Thread-3    ] [DEBUG]  coinjoining 16410404 satoshi
Exception in thread Thread-3:
Traceback (most recent call last):
  File "C:\TOOLS\Python27\lib\threading.py", line 810, in __bootstrap_inner
    self.run()
  File "tumbler.py", line 320, in run
    self.init_tx(tx, self.balance_by_mixdepth[tx['srcmixdepth']], sweep)
  File "tumbler.py", line 280, in init_tx
    self.create_tx()
  File "tumbler.py", line 229, in create_tx
    cj_amount, self.tx['makercount'])
  File "tumbler.py", line 142, in tumbler_choose_orders
    self.ignored_makers + active_nicks)
  File "C:\TOOLS\JOINMARKET\joinmarket\support.py", line 247, in choose_orders
    'counterparty'] not in ignored_makers]
  File "C:\TOOLS\JOINMARKET\joinmarket\support.py", line 168, in calc_cj_fee
    real_cjfee = int(cjfee)
ValueError: invalid literal for int() with base 10: '5E-9'
```
Thanks

**Comment by AdamISZ** [comment]
This is the fault of one of the makers in the transaction, who's publishing an absorder with a non-integer amount (should be in satoshis).

 We should be catching this exception, adding the maker to the ignored_makers list and retrying the transaction.

Thanks for the report.

**Comment by solsimpais** [comment]
Thanks, I have changed joinmarket/taker.py as you have suggested and tumbling is on its way! I just get:a few such messages along the way:
```
2016-01-14 17:07:01,244 [MainThread  ] [DEBUG]  Got non integer coinjoin fee: 8783 for an absorder from Fixolegib
```

**Comment by raedah** [comment]
This seems to be blocking the orderbook table from updating at all.
Sending !cancel 0 to the pit from any user generates:
[DEBUG]  !cancel ValueError("invalid literal for int() with base 10: '5E-9'",)
This makes me believe existing users are no longer able to track valid orderbook updates.
A fix will need to make it to master.

**Comment by AdamISZ** [comment]
Fix applied to master, see tag v0.1.1
